deploy/charts/fedlearner-stack/Chart.yaml

```diff
@@ -19,3 +20,8 @@ dependencies:
 - name: argo
   repository: https://argoproj.github.io/argo-helm
   version: 0.9.4
+  condition: argo.enabled
+- name: mariadb
+  repository: ""
+  version: 7.5.1
```

Comment:
consider using version `10.0.38`, 'caz it's fully [compatible](https://mariadb.com/kb/en/mariadb-vs-mysql-compatibility/#replication-compatibility) with MySQL 5.6 including [performance schema](https://mariadb.com/kb/en/performance-schema-overview/), which is also supported on internal RDS and public RDS

Reply:
This is the helm app version not mariadb version. The mariadb version is 10.3.23